installation/on_alpine_linux.md

```diff
@@ -0,0 +1,12 @@
+# On Alpine Linux
+
+Alpine Linux includes the Crystal compiler in the community repository, starting from version Alpine 3.7.
+
+Currently, the package is only available for `aarch64` and `x86_64` platforms.
+
+## Install
+
+```bash
+sudo apk update
+sudo apk add crystal shards
```

Comment:
I'd omit sudo. It's not required to run apk via sudo. It must be run as root, but there are many ways how to achieve it and sudo is not even part of the Alpine base.